strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/configuration/ConfigurationManagerTest.java

```diff
@@ -66,21 +67,40 @@
 
     private YAMLMapper yamlMapper;
 
-
     @BeforeEach
     public void setUp()
```

Comment:
@ptirador will be better to use `TestInfo` here as well, currently its not thread safety.